src/Hintman/Core/PrInfo.hs

```diff
@@ -1,22 +1,37 @@
 -- | Core types related to pull request.
 
 module Hintman.Core.PrInfo
-       ( Repo (..)
+       ( PrInfo (..)
        , Owner (..)
+       , Repo (..)
+       , Branch (..)
        , PrNumber (..)
        ) where
 
 import Servant (ToHttpApiData (..))
+import Text.Diff.Parse.Types (FileDeltas)
 
 
-newtype Repo = Repo
-    { unRepo :: Text
-    } deriving newtype (ToHttpApiData)
+-- | Data type that represents the current pull request.
+data PrInfo = PrInfo
+    { prInfoOwner  :: !Owner
+    , prInfoRepo   :: !Repo
+    , prInfoBranch :: !Branch
```

Comment:
Could we also put `PrNumber` in this data type? It might not be useful for `hlint` but will be useful for `submitComment` function...